src/modules/logger/logger.cpp

```diff
@@ -730,11 +743,28 @@ void Logger::run()
 			param_get(_sdlog_mode_handle, &_sdlog_mode);
 		}
 
-		if (_sdlog_mode == 3) {
-			add_calibration_topics();
+		if (_sdlog_profile_handle != PARAM_INVALID) {
+			param_get(_sdlog_profile_handle, &_sdlog_profile);
```

Comment:
`_sdlog_profile` does not need to be a class attribute since it's only used here.